backend/ethereum/channel/contractbackend.go

```diff
@@ -122,6 +128,13 @@ func (c *ContractBackend) pastOffsetBlockNum(ctx context.Context) (uint64, error
 // set the value manually afterwards if it should be different from 0.
 func (c *ContractBackend) NewTransactor(ctx context.Context, gasLimit uint64,
 	acc accounts.Account) (*bind.TransactOpts, error) {
+	c.nonceMtx.Lock()
+	defer c.nonceMtx.Unlock()
+	expectedNextNonce, found := c.expectedNextNonce[acc.Address]
```

Comment:
You use the `expectedNextNonce` value below. Does golang define how it is set if the map access fails?

Reply:
[Default zero value](https://golang.org/ref/spec#The_zero_value)

Reply:
Good to know.